src/js/utils/dom.ts

```diff
@@ -0,0 +1,12 @@
+export const $ = (
+  selector: string,
+  target: Document | HTMLElement = document
+): HTMLElement => target.querySelector(selector) as HTMLElement;
+
+export const $$ = (
+  selector: string,
+  target: Document | HTMLElement = document
+): NodeListOf<HTMLElement> => target.querySelectorAll(selector);
```

Comment:
```suggestion
): HTMLElement[] => [ ...target.querySelectorAll(selector) ];
```
이렇게 배열로 만들어지기 때문에 더 다양하게 응용할 수 있답니다!